diff --git a/plugins/views/views_php_handler_field.inc b/plugins/views/views_php_handler_field.inc
index 9164d9a..f7d1f5c 100644
--- a/plugins/views/views_php_handler_field.inc
+++ b/plugins/views/views_php_handler_field.inc
@@ -210,8 +210,8 @@ class views_php_handler_field extends views_handler_field {
     // Ecexute output PHP code.
     if (!empty($this->options['php_output']) && isset($this->php_output_lamda_function)) {
       $normalized_row = new stdClass;
-      foreach ($this->view->display_handler->get_handlers('field') as $field => $handler) {
-        $normalized_row->$field = isset($values->{$handler->field_alias}) ? $values->{$handler->field_alias} : NULL;
+      foreach ($this->view->style_plugin->rendered_fields{$this->view->row_index} as $field => $value) {
+        $normalized_row->$field = $value;
       }
 
       $function = $this->php_output_lamda_function;
